velox/expression/VectorWriters.h

```diff
@@ -532,7 +532,14 @@ struct VectorWriter<Generic<T>> : public VectorWriterBase {
       castWriter_->ensureSize(size);
     } else {
       if (vector_->size() < size) {
-        vector_->resize(size, false);
+        if (vector_->typeKind() == TypeKind::ROW) {
+          // Avoid calling resize on all children by adding top level nulls
+          // only.
+          vector_->asUnchecked<RowVector>()->appendNulls(
```

Comment:
do we ensure we reset the null bit when we write a value to this position in the VectorWriter?

Reply:
yes